### PR TITLE
add GraphQL

### PR DESCRIPTION
**Pull request by rvantonder**
> “You have one go at life. One chance to follow your dreams. Don't waste it on planning your life, spend it on ~doing.~ fixing GQL highlighting.  
>   -- ~Emily Huffstetler~ @rvantonder

Sorry not sorry about the table formatting, it was a bit ugly and my editor auto-corrected it.

### CODE REVIEW

**Comment by rvantonder** [comment]
Btw tested manually on `schema.graphql` and stuff 

Screen Shot 2020-10-16 at 2 45 54 PM

File: GraphQL/GraphQL.tmLanguage [hunk 1]
```diff
@@ -0,0 +1,1889 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (slimsag) [inline GraphQL/GraphQL.tmLanguage:1]: Will need to convert this to a .sublime-syntax file following the 2nd bullet point here: https://github.com/sourcegraph/syntect_server#adding-languages

**Comment by slimsag** [comment]
This rocks! will merge and push this through the rest of the pipes later today

File: GraphQL/GraphQL.tmLanguage (hunk 1 above)
Comment (rvantonder) [reply GraphQL/GraphQL.tmLanguage:1]: OK the tool I used created a [bogus `sublime-syntax` file](https://github.com/rvantonder/gql-sublimetext-yolo/commit/96801f915628f643c0c7c1e558c9f7637ccf17dc#diff-2ddf8fc34760839518ccde855507781d583b091e0316220da49cc0edf8625cb9) from the original JSON and didn't work with sublime, but the `.tmLanguage` did. Sounds like this ^ is a better way that may properly convert a `sublime-syntax` file. will do.

Comment (rvantonder) [reply GraphQL/GraphQL.tmLanguage:1]: OK I couldn't do it directly but by sort of exhaustively calling convert commands I think I ended up with a `.sublime-syntax` version.

**Comment by rvantonder** [comment]
I had some trouble converting to `sublime-syntax` until I realized the definition file had some [garbage](https://github.com/rvantonder/gql-sublimetext-yolo/commit/1bb438c8d4ed9e30820da5e600149285723758d8) in it. Fixed that and things are working.

File: GraphQL/tests/syntax_test_basic.graphql [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# SYNTAX TEST "Packages/GraphQL/GraphQL.sublime-syntax"
```
Comment (slimsag) [inline GraphQL/tests/syntax_test_basic.graphql:1]: I think this test is failing due to some kind of linkage error - but we don't need to duplicate the tests here anyway so I will just remove this file. Better to have these upstream in the actual SOURCE repo :)

**Review by slimsag — APPROVED** [review]

**Pull request merged**